README.md

```diff
@@ -47,6 +47,11 @@ AUTH_PASSWORD=any_password
 
 - `poetry run pytest`
 
+### Load tests
```

Comment:
Which server do these tests use?

Reply:
The server you want. Running locust opens a web ui where you can specify stuff like duration, number of users, and the server you want to test. So you can test both deployment and local servers without having to alter the code